frontend/components/notifications/management/modal/NotificationsManagementModalDashboardsDelete.vue

```diff
@@ -0,0 +1,85 @@
+<script lang="ts" setup>
+import type { NotificationSettingsDashboardsTableRow } from '~/types/api/notifications'
+
+const { t: $t } = useTranslation()
+const {
+  close,
+  props,
+} = useBcDialog<NotificationSettingsDashboardsTableRow>()
+
+const idCancel = useId()
+const lastFocusedElement = ref()
+onBeforeMount(() => {
+  lastFocusedElement.value = document.activeElement
+})
+onMounted(() => {
+  document.getElementById(idCancel)?.focus()
+})
+onUnmounted(() => {
+  // focus management with `primevues dialog` is not working properly
+  lastFocusedElement.value?.focus()
+})
+const emit = defineEmits<{
+  (e: 'delete',
+    payload: Pick<
+      NotificationSettingsDashboardsTableRow,
+      | 'dashboard_id'
+      | 'group_id'
+      | 'is_account_dashboard'
+      | 'settings'
+    >
+  ): void,
+}>()
+const handleDelete = () => {
+  if (props.value?.dashboard_id && props.value?.group_id) {
+    emit('delete', {
+      dashboard_id: props.value.dashboard_id,
+      group_id: props.value.group_id,
+      is_account_dashboard: props.value.is_account_dashboard,
+      settings: props.value.settings,
+    })
+  }
+  close()
+}
+</script>
+
+<template>
+  <div
+    v-focustrap
+    @keydown.esc.stop="close"
+  >
+    <!-- <pre>
+    {{ props }}
+  </pre> -->
+    <BcText tag="h2" variant="lg">
+      Delete all notifications
+    </BcText>
+    <p class="notifications-management-modal-dashboards-delete__content">
+      Do you really want to delete all notifications for this dashboard?
```

Comment:
Please, add translations. 
```suggestion
      Do you really want to delete all notifications for this dashboard?
```